File: blog.py
# 型指定と出力、型確認
print("型指定と出力、型確認")
num = 42

print("改行あり")
print(num)
print(type(num))
print() # 改行

# 改行あり
print("改行あり")
print(42)
print(42)
print() # 改行

# 改行なし
print("改行なし")
print(42, end="")
print(42, end="")
print("\n") # 改行

# フォーマット済み文字列
print("フォーマット済み文字列")
age = 42
print("父は{}歳である".format(age))
print(f"父は{age}歳である")
print("\n") # 改行

# キーボードからの入力を取得
print("キーボードからの入力を取得")
key = input()
print(key)

File: blog.go
package main

import (
	"fmt"
	"reflect"
)

func main() {
	fmt.Println("型指定と出力、型確認")
	var num int = 42
	fmt.Println(num)
	fmt.Println(reflect.TypeOf(num))

	// 改行あり
	fmt.Println() //改行
	fmt.Println("改行あり")
	fmt.Println(42)
	fmt.Println(42)

	// 改行なし
	fmt.Println() //改行
	fmt.Println("改行なし")
	fmt.Print(42)
	fmt.Print(42)

	// フォーマット済み文字列
	fmt.Println("\n") //改行
	fmt.Println("フォーマット済み文字列")
	var age int = 42
	fmt.Printf("父は%d歳である\n", age)

	// キーボードからの入力を取得
	fmt.Println() //改行
	var key int
	fmt.Scan(&key)
	fmt.Println(key)
}
